lib/node/NodeSourcePlugin.js

```diff
@@ -37,33 +38,39 @@ NodeSourcePlugin.prototype.apply = function(compiler) {
 			if(localOptions.process) {
 				var processType = localOptions.process;
 				parser.plugin("expression process", function() {
-					return ModuleParserHelpers.addParsedVariable(this, "process", "require(" + JSON.stringify(getPathToModule("process", processType)) + ")");
+					return ModuleParserHelpers.addParsedVariable(this, "process", "require(" +
+					    JSON.stringify(path.relative(this.state.module.context, getPathToModule("process", processType))) + ")");
```

Comment:
Ok, I'm sure I can eliminate every test error by write the declaration of `Buffer`, `setImmediate`, `clearImmediate` before their usage in `webpack/node_modules/buffer/index.js` and `webpack/node_modules/timers-browserify/main.js`